package/scripts/postinstall

```diff
@@ -57,7 +57,7 @@ then
   if [[ -f "/usr/local/bin/brew" && -d "/usr/local/Homebrew" ]]
   then
     cp -pRL "${homebrew_directory}/.git" "/usr/local/Homebrew/"
-    mv "${homebrew_directory}/cache_api" "/usr/local/Homebrew/"
+    mv -n "${homebrew_directory}/cache_api" "/usr/local/Homebrew/"
```

Comment:
@nstrauss Why is this needed? This directory should never exist already, no?

Reply:
Ok, might be worth doing `rm -rf` instead to handle that case rather than avoid copying new data?

Reply:
Choosing to remove the `-n` option rather than risk stale data. Tested again and couldn't replicate the same failure path. I think it was originally due to the git safe directory stuff. Seeing as it's an edge case, and maybe not even accessible anymore, fine with going back to bare `mv`. https://github.com/Homebrew/brew/pull/16073/commits/8560827cdb0949a34ff701ea317c89328c4afd88